### ISSUE TITLE
[Home] Form control does not have a corresponding label

### ISSUE THREAD

**Issue by degraphic** [veshu/accessibility-bughunt-bluesource#88]
<h2>Steps to reproduce the behavior</h2>
1. Go to Wellmark app. or http://localhost:4200/home<br>
2. Then Click  WAVE Web Accessibility Tool from Extention On Chrome

<h2>Actual behavior</h2>
We can see Form control does not have a corresponding label inside the form

<h2>Expected behavior</h2>
I think we can add Label inside the form to fix the error

**Device**: Laptop MacbookPro
**Operating System**: MacOs Mojave 10.14.6
**Browser(with version)**: Chrome Version 78.0.3904.70 (Official Build) (64-bit)

**WCAG Category**: 2. Operable
**WCAG Level**: A
**WCAG Success Criterion**: 1.1.1 Non text content
**Tools used**: WAVE

**Screenshoot:**
<br>

![Screen Shot 2019-11-01 at 01 47 15](https://user-images.githubusercontent.com/4364214/67977358-83c4c100-fc4a-11e9-9e05-6f321ba8ba15.png)
<br>
![Screen Shot 2019-11-01 at 01 51 54](https://user-images.githubusercontent.com/4364214/67977359-845d5780-fc4a-11e9-9e47-507ec0ac34d9.png)

**Comment by drasticdpk** [comment]
WCAG Category: 1. Perceivable
